blocks/callout/callout.css

```diff
@@ -168,8 +170,7 @@
 .callout > div > .callout-content {
           a.button:hover {
             text-decoration: none;
-            border-color: var(--light-grey);
-            color: var(--light-grey);
+            color: var(--new-green);
```

Comment:
In case the callout block is yellow, we should use black instead of green on hover and border. Please see the customer requirement [here](https://github.com/aemsites/ingredion/issues/321#issuecomment-2919749280).